.pre-commit-config.yaml

```diff
@@ -18,3 +18,4 @@ repos:
         entry: make
         args: ["lint"]
         types: ["go"]
+        pass_filenames: false
```

Comment:
`make lint` was being called for every file on commit. This makes it so its only called once.